lib/importer/govuk-prototype-kit.config.json

```diff
@@ -53,6 +53,10 @@
     {
       "macroName": "importerTableView",
       "importFrom": "importer/macros/table_view.njk"
+    },
+    {
+      "macroName": "importerRowSelector",
```

Comment:
Does it only select whole rows? Or is it a "range selector"?

Reply:
Range, will rename

Reply:
Fixed in 41619a7